LoopKitUI/ViewModels/DisplayGlucosePreference.swift

```diff
@@ -13,13 +13,17 @@ import LoopKit
 
 public class DisplayGlucosePreference: ObservableObject {
     @Published public private(set) var unit: HKUnit
+    @Published public private(set) var rateUnit: HKUnit
     @Published public private(set) var formatter: QuantityFormatter
     @Published public private(set) var minuteRateFormatter: QuantityFormatter
 
     public init(displayGlucoseUnit: HKUnit) {
+        let rateUnit = displayGlucoseUnit.unitDivided(by: .minute())
+
         self.unit = displayGlucoseUnit
+        self.rateUnit = rateUnit
```

Comment:
shouldn't this be a computed property? Or is there a need that the `rateUnit` does not have the same base unit as `unit`?